README.md

```diff
@@ -431,9 +456,9 @@ optional arguments:
 - [ ] Plugin django app is defined using PulpAppConfig as a parent
 - [ ] Plugin entry point is defined
 - [ ] Necessary models/serializers/viewsets are defined. At a minimum:
-    - [ ] models for plugin content type, remote, publisher
-    - [ ] serializers for plugin content type, remote, publisher
-    - [ ] viewset for plugin content type, remote, publisher
+    - [ ] models for plugin content type, repository, remote, publisher
```

Comment:
We don't have publisher's anymore technically right? Should that part be removed?